files/image_config/resolv-config/resolv.conf.j2

```diff
@@ -0,0 +1,3 @@
+{% for ip in DNS_NAMESERVER|sort %}
+nameserver {{ ip }}
```

Comment:
>nameserver [](http://example.com/codeflow?start=0&length=10)

In some use case, people are using "/etc/resolv.conf" to manually config DNS entries. In this PR, the content will be overwrite by j2 rendering.

Could you still allow old use case?

Reply:
Those are mutually exclusive approaches. If the user wants to edit /etc/resolv.conf manually it should not configure DNS via config DB and vice-versa. The default behavior is preserved if no DNS entries are specified in the config DB.